### PR TITLE
Tell students to set their name/email in Git

### PR DESCRIPTION
**Pull request by gabebw**
We have dummy values set in Vagrant so it won't crash if this isn't done, but
it'll make their commits look better.

Fixes #36.

### CODE REVIEW

File: sites/en/installfest/set_up_virtual_machine.md [hunk 1]
```diff
@@ -105,6 +105,22 @@ You will see a welcome message something like this:
 
     RailsBridge-VM:~/workspace$
 
+## Set up git
+
+Let's set up Git, a way to track the code that you write tomorrow.
+
+Type this, replacing `"Your Name"` with your full name in quotes:
+
+    git config user.name "Your Name"
```
Comment (gabebw) [inline sites/en/installfest/set_up_virtual_machine.md]: This should be `--global`

**Comment by gabebw** [comment]
@decklin Can I merge this?

**Comment by mxie** [comment]
@gabebw Go for it.

**Comment by decklin** [comment]
Sorry! Holidays.

**Pull request merged**